spec/index.bs

```diff
@@ -2517,6 +2518,82 @@ dictionary GPUImageBitmapCopyView {
     </div>
 </div>
 
+### <dfn method for=GPUCommandEncoder>copyBufferToTexture(source, destination, copySize)</dfn> ### {#GPUCommandEncoder-copyBufferToTexture}
+
+<div algorithm="GPUCommandEncoder.copyBufferToTexture">
+
+  **Arguments:**
+    - {{GPUBufferCopyView}} |source|
+    - {{GPUTextureCopyView}} |destination|
+    - {{GPUExtent3D}} |copySize|
+
+  **Returns:** void
+
+  Encode a command into the {{GPUCommandEncoder}} that copies |copySize| texels from a sub-region of a {{GPUBuffer}} |source| to another sub-region of a {{GPUTexture}} [=subresource=] |destination|.
```

Comment:
reading long lines in github review is hard, unfortunately

Reply:
Does it look better now?